packages/generator-electrode/generators/app/templates/_package.json

```diff
@@ -31,9 +31,9 @@
   },
   "dependencies": {
     "bluebird": "^3.4.6",
-    "electrode-archetype-react-app": "^4.0.0",
+    "electrode-archetype-react-app": "^5.0.0",
```

Comment:
Pending on the releases of "electrode-archetype-react-app": "^5.0.0" and "electrode-react-webapp": "^3.0.0".